Jenkinsfile

```diff
@@ -57,32 +57,14 @@ pipeline {
                 }
             }
         }
-        stage("Env dev namespace") {
-            agent {
-                node {
-                    label "master"
-                }
-            }
-            when {
-              expression { GIT_BRANCH ==~ /(.*develop.*|.*feature.*)/ }
-            }
-            steps {
-                script {
-                    // Arbitrary Groovy Script executions can do in script tags
-                    env.PROJECT_NAMESPACE = "${NAMESPACE_PREFIX}-dev"
-                    env.NODE_ENV = "dev"
-                    env.REACT_APP_BACKEND_URI = "https://omp-backend-omp-dev.apps.s11.core.rht-labs.com"
-                }
-            }
-        }
         stage("Ansible") {
             agent {
                 node {
                     label "jenkins-slave-ansible"
                 }
             }
             when {
-              expression { GIT_BRANCH ==~ /(.*master|.*develop.*|.*feature.*)/ }
+              expression { GIT_BRANCH ==~ /(.*master)/ }
```

Comment:
We should just assume master. These when blocks aren't needed. This block and the ones below